Mathlib/RingTheory/Polynomial/SeparableDegree.lean

```diff
@@ -83,6 +84,7 @@ theorem HasSeparableContraction.dvd_degree : hf.degree ∣ f.natDegree :=
   Dvd.intro (q ^ a) ha
 
 /-- In exponential characteristic one, the separable degree equals the degree. -/
+@[stacks 09H5 "third part"]
```

Comment:
I don't think this tag is right.

Reply:
I see. This lemma corresponds to the content written in the small paragraph right following 09H5.

```
The separable degree of P always divides the degree and the quotient is a power of the characteristic. If the characteristic is zero, then degs(P)=deg(P).
```

There is no tag in Stacks Project for this small paragraph. I'll just remove this tag.